### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by alshakero**
Because I'm used to working alone on this most of the time. I didn't pull before working so some of @warpech changes weren't released in 3.2.0. But I'll release just for them after this review.

I released 3.2.0 before this PR because I forgot about the new restrictions of pushing to master. 

What's new?
1. It now runs inside ShadowDOM to avoid CSS and element ID's collisions.
2. It now has beta health checks tab.
3. CSS is enhanced a little.

Fixes https://github.com/Starcounter/DevTools/issues/5 & https://github.com/Starcounter/DevTools/issues/71

### CODE REVIEW

**Comment by warpech** [comment]
You have committed 2 zip files in https://github.com/Starcounter/DevTools/pull/72/commits/1bd698ce8316e575d496e6539303e1305e1ca94f. I guess that's unintended?

File: src/views/health-checks.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<template>
+    <div class="tab">
+        <div v-if="compatible">
+            <h1>Health checks</h1>
+            <p>This is a baby tab. Currently it only checks if all the Custom Elements you're using are imported. Soon we'll add more health checks you can use for your apps.
```
Comment (warpech) [inline src/views/health-checks.vue]: I don't know what a baby tab is :) Do you mean "beta feature"?

File: src/views/health-checks.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+var currentImports = [];
+
+export default {
+  name: 'health-checks',
+  props: ['overlay'],
+  data() {
+    return {
+      compatible:
+        this.getCurrentWindow().Polymer &&
+        this.getCurrentWindow().Polymer.version.startsWith('2'), // only works with Polymer 2
```
Comment (warpech) [inline src/views/health-checks.vue]: The recommended way to test for Polymer 2+ is to check if `Polymer.Element` is defined.

Source: https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements

File: src/views/health-checks.vue [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      }
+      return currentWindow;
+    },
+    checkElementsHealth() {
+      this.customElementsHealth = findUndefinedElements(this.getCurrentWindow());
+    }
+  }
+};
+</script>
+<style scoped>
```
Comment (warpech) [inline src/views/health-checks.vue]: `scoped` attribute is not implemented by any browser anymore.

Comment (alshakero) [reply src/views/health-checks.vue]: Yes but here it's a Vue feature.

File: src/views/health-checks.vue (hunk 1 above)
Comment (alshakero) [reply src/views/health-checks.vue]: I mean a tab that should be extended with further checks, WDYT we should call it?

**Comment by alshakero** [comment]
> How is it possible without a revert commit bettwen them?

Haha you know git. What happened is that your changes don't conflict with mine **only in `src` folder** so my changes could be committed after yours. THEN I built and now your changes are in `dist`. So they tricked you into believing they happened twice.

**Comment by warpech** [comment]
That I would understand, but I can see that even for the changes in `README.md` file. This file is not in `build`.

File: src/views/health-checks.vue (hunk 3 above)
Comment (warpech) [reply src/views/health-checks.vue]: Did not see that coming! Thanks for the explanation

File: src/views/health-checks.vue (hunk 1 above)
Comment (warpech) [reply src/views/health-checks.vue]: I think it is fine now, thanks

**Pull request merged**